R/run_stress_test.R

```diff
@@ -36,6 +36,58 @@ run_stress_test <- function(asset_type,
                             shock_year = 2030,
                             term = 2,
                             company_exclusion = TRUE) {
+  warn_if_asset_type_is_longer_than_1(asset_type)
+  matched_asset_type <- match.arg(asset_type)
+
+  # Allow testing anything before the long computation that follows
+  if (is_dev_env()) {
+    return(invisible(matched_asset_type))
+  }
+
+  # FIXME: Avoid clutter from printed "messages" that the user can't suppress
+  x <- capture.output(
+    run_stress_test_impl(
+      asset_type = matched_asset_type,
+      lgd_senior_claims = lgd_senior_claims,
+      lgd_subordinated_claims = lgd_subordinated_claims,
+      terminal_value = terminal_value,
+      risk_free_rate = risk_free_rate,
+      discount_rate = discount_rate,
+      div_netprofit_prop_coef = div_netprofit_prop_coef,
+      shock_year = shock_year,
+      term = term,
+      company_exclusion = company_exclusion
+    )
+  )
+
+  invisible(asset_type)
```

Comment:
Maybe a more useful output would be the input data -- which suggests that maybe the first argument should be `data` and take an R object (e.g. a list of dataframes, or a nested dataframe) or a path to a folder with the expected input data. For backward compatibility default would be `data = Sys.getenv("ST_SUCH_AND_SUCH")`.

Reply:
I would not see a benefit in returning anything other than  NULL/invisible(asset_type) here.